demo/capture_Snakefile.smk

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env snakemake
+
+'''
+This Snakefile is made to run all the modules compatible with Capture workflow.
+Compatibility of a workflow can be checked by referring to the pairing_config parameter present in a default.yaml file of that module.
+'''
+##### SETUP #####
+
+import oncopipe as op
+
+SAMPLES = op.load_samples("data/samples.tsv")
+CAPTURE = op.filter_samples(SAMPLES, seq_type = "capture")
+
+
+##### REFERENCE_FILES WORKFLOW #####
+
+
+subworkflow reference_files:
+    workdir:
+        "reference/"
+    snakefile:
+        "../workflows/reference_files/2.4/reference_files.smk"
+    configfile:
+        "../workflows/reference_files/2.4/config/default.yaml"
+
+
+##### CONFIGURATION FILES #####
+
+
+# Load module-specific configuration
+configfile: "../modules/slms_3/1.0/config/default.yaml"
+configfile: "../modules/sage/1.0/config/default.yaml"
+configfile: "../modules/utils/2.1/config/default.yaml"
+configfile: "../modules/picard_qc/1.0/config/default.yaml"
+configfile: "../modules/bam2fastq/1.2/config/default.yaml"
+configfile: "../modules/manta/2.3/config/default.yaml"
+configfile: "../modules/gridss/1.1/config/default.yaml"
+configfile: "../modules/sequenza/1.4/config/default.yaml"
+configfile: "../modules/strelka/1.1/config/default.yaml"
+configfile: "../modules/bwa_mem/1.1/config/default.yaml"
+configfile: "../modules/lofreq/1.0/config/default.yaml"
+configfile: "../modules/starfish/2.0/config/default.yaml"
+configfile: "../modules/battenberg/1.1/config/default.yaml"
+configfile: "../modules/mutect2/2.0/config/default.yaml"
+configfile: "../modules/varscan/1.1/config/default.yaml"
+configfile: "../modules/liftover/1.2/config/default.yaml"
+
+# Load project-specific config, which includes the shared 
+# configuration and some module-specific config updates
+configfile: "capture_config.yaml"
+
+
+##### CONFIGURATION UPDATES #####
+
+
+# Use all samples as a default sample list for each module
+config["lcr-modules"]["_shared"]["samples"] = CAPTURE
+
+##### MODULE SNAKEFILES #####
+
+
+# Load module-specific snakefiles
+include: "../modules/slms_3/1.0/slms_3.smk"
+include: "../modules/sage/1.0/sage.smk"
+include: "../modules/utils/2.1/utils.smk"
+include: "../modules/picard_qc/1.0/picard_qc.smk"
+include: "../modules/manta/2.3/manta.smk"
+include: "../modules/sequenza/1.4/sequenza.smk"
+include: "../modules/strelka/1.1/strelka.smk"
+include: "../modules/bwa_mem/1.1/bwa_mem.smk"
+include: "../modules/gridss/1.1/gridss.smk"
+include: "../modules/bam2fastq/1.2/bam2fastq.smk"
+include: "../modules/lofreq/1.0/lofreq.smk"
+include: "../modules/starfish/2.0/starfish.smk"
+include: "../modules/battenberg/1.1/battenberg.smk"
+include: "../modules/mutect2/2.0/mutect2.smk"
+include: "../modules/varscan/1.1/varscan.smk"
+include: "../modules/liftover/1.2/liftover.smk"
+
+
+
+##### TARGETS ######
+
+rule all:
+    input:
+        rules._lofreq_all.input,
```

Comment:
can you please, in each snakefile for all 3 workflows, organize the module `include`,  `config`, and rule all in the same order? They are currently all mixed, and I know it is like that in master, but it will help to have the snakefile more organized and easier to follow

Reply:
certainly

Reply:
done